doc/bus.rst

```diff
@@ -8,13 +8,20 @@ The bus provides a wrapper around a physical or virtual CAN Bus.
 
 A thread safe bus wrapper is also available, see `Thread safe bus`_.
 
+Autoconfig Bus
```

Comment:
Why moving this autoconfig bus before the `BusABC`?

Reply:
Because that is the main entry point into the library if you want to communicate.

Reply:
Well it is a wrapper/constructor around a concrete implementation of the `BusABC` - both should be covered and I assumed that the more interesting/important item was the class that acts as the primary interface. 

Leave your changes - we should just add a clarifying note to the top to help explain the class structure to users.